app/controllers/pages_controller.rb

```diff
@@ -1,4 +1,36 @@
+# frozen_string_literal: true
+
 class PagesController < ApplicationController
-  def home
+  include HTTParty
+
+  def home; end
+
+  def init_data
+    api_url = "#{ENV['CLOVERPOP_DOMAIN']}/api/v1/decisions"
```

Comment:
@SerhiiBorozenets It lokks like IOt would be better to move it to constant controller constant with freezing constant